interface/reports/index.php

```diff
@@ -3,7 +3,7 @@
 require_once (dirname(__FILE__) . "/../globals.php");
 require_once (dirname(__FILE__) . "/../../library/sql.inc");
 require_once (dirname(__FILE__) . "/../../library/Smarty.class.php");
-require_once (dirname(__FILE__) . "/../../library/adodb/adodb-pager.inc.php");
```

Comment:
If you are calling globals.php, you can use the /library /assets and /modules global variables in the include names.  These are:

-  $GLOBALS['srcdir'], 
- $GLOBALS['assets'],
- $GLOBALS['modules_dir']

There is also $GLOBALS['standard_js_path'] for /assets/js standard third party javascript libraries.

Then you can call $GLOBALS['srcdir'] . 'headers.inc.php'  (or use $srcdir) to be able to call the functions in there like function call_required_libraries($library_array).
These functions may be expanded on to allow different parameters to be used as you require.
For instance, the adodb includes, and even sql.inc are not yet added to that header file.
  This is all very functional at the moment, but since you are doing it, bringing it absolutely up to the current consistency standards would be very nice.  It also isn't a bad time (especially as a series of commits) to do the code changes (as you have) then do the directory changes (as you have), then finally do any white-space adjustments within the files.

Finally, please please go ahead and add yourself to any attributions you deem appropriate.  It is good to be able to quickly see who "owns" the latest iterations.

Reply:
So the directory paths need to be changed to include the $GLOBALS variable, right? Is there anything else I need to do? Also what d'you mean by adding myself to any attributions? Didn't quite understand that.

Reply:
It would be nice to do the directory changes, add common libraries to the headers functions etc...  Mostly, we are just waiting on someone else to review this...  I am approving it as-is now in hopes it attracts some attention.

Reply:
@aethelwulffe Which part of the application does the reports/index.php file govern? I can't seem to figure it out to test whether changing the directories breaks anything.